freqtrade/exchange/exchange.py

```diff
@@ -1686,10 +1686,14 @@ def fetch_positions(self, pair: str | None = None) -> list[CcxtPosition]:
         if self._config["dry_run"] or self.trading_mode != TradingMode.FUTURES:
             return []
         try:
+            positions: list[CcxtPosition]
             symbols = []
             if pair:
                 symbols.append(pair)
-            positions: list[CcxtPosition] = self._api.fetch_positions(symbols)
+            if symbols:
+                positions = self._api.fetch_positions(symbols)
+            else:
+                positions = self._api.fetch_positions()
             self._log_exchange_response("fetch_positions", positions)
```

Comment:
Any particular reason to have this? 
This seems to be a pointless change - as ccxt does support passing in an empty list / none.

Reply:
Because i found that bitget to call fetch_positions has some problem if symbols = [],
ccxt/bitget.py
```pyhon
   def fetch_positions(self, symbols: Strings = None, params={}) -> List[Position]:  
	...
        if symbols is not None:
            first = self.safe_string(symbols, 0)
            sandboxMode = self.safe_bool(self.options, 'sandboxMode', False)
            if sandboxMode:
                sandboxSymbol = self.convert_symbol_for_sandbox(first)
                market = self.market(sandboxSymbol)
            else:
                market = self.market(first)`
	....
```
_symbols is not None_  widely used in other exchanges in ccxt
So I think _self._api.fetch_positions()_ might be better than _self._api.fetch_positions([])_.

Reply:
If ccxt has a problem with this on bitget, please open an issue with ccxt.

It's working for **all** other exchanges that support futures - we'll not make an exception for bitget just because of a bug in ccxt.